pkg/lvmdriver/services/identity_service.go

```diff
@@ -0,0 +1,89 @@
+package services
+
+import (
+	"context"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/wrapperspb"
+	"k8s.io/klog/v2"
+)
+
+// IdentityService handles requests from the container orchestrator
+// to determine readiness, capabilities, and identify the driver
+type IdentityService struct {
+	csi.UnimplementedIdentityServer
+	name         string
+	version      string
+	capabilities []csi.PluginCapability_Service_Type
+	ready        func() (bool, error)
+}
+
+// NewIdentityService returns a new IdentityService.
+//
+// ready is a function to check the plugin status.
+// It should return non-nil error if the plugin is not healthy.
+// If the plugin is not yet ready, it should return (false, nil).
+// Otherwise, return (true, nil).
+func NewIdentityService(name string, version string, ready func() (bool, error)) csi.IdentityServer {
+	return &IdentityService{
+		ready:   ready,
+		name:    name,
+		version: version,
+		capabilities: []csi.PluginCapability_Service_Type{
+			csi.PluginCapability_Service_UNKNOWN,
+		},
+	}
+}
+
+func (s IdentityService) GetPluginInfo(ctx context.Context, req *csi.GetPluginInfoRequest) (*csi.GetPluginInfoResponse, error) {
+	klog.V(2).Info("received PluginInfoRequest")
```

Comment:
```suggestion
	klog.V(1).Info("received PluginInfoRequest")
```
I think its enough to have one level of debug logs tbh, There is also a recommendation to not use too many of them as it will be hard to identify the log topology later on

Reply:
I think I'd rather have 2 levels at least. Level 1 being critical logs that will actually be helpful if customers need to debug and then another level we can rely on for more information on data flow